### DISCUSSION TITLE
How to reverse the data so that latest item is first one

### DISCUSSION THREAD

**Discussion by LavaGolem** [prabhuignoto/react-chrono#399] [Q&A]
Hi ppl, 

I need to show the latest items first in the timeline, but even if I reverse the data before I pass it to the timeline, it's still displayed oldest item first

Thank you for yours help

**Comment by prabhuignoto** [comment]
@LavaGolem the timeline component honors the order of timeline items in the collection. here is an example of the timeline rendered in reverse https://codesandbox.io/s/react-chrono-vertical-basic-forked-xq9msr

do check out and let me know if this works for you
